scripts/garage

```diff
@@ -14,27 +11,32 @@ def cli():
 
 @cli.command()
 @click.option(
-    '--snapshot_dir',
-    required=True,
-    help='Directory of the pickle file to resume experiment from.')
+    '--resume_from_dir', required=True, help='Path to resume experiment from.')
 @click.option(
-    '--resume_epoch',
+    '--resume_from_epoch',
```

Comment:
this command will appear as `garage resume` so i think `--from_epoch` and `--from_dir` are still good arguments names, e.g.

`garage resume --from_dir=/data/foo/bar --from_epoch=last`

Reply:
Yeah, I agree that this would be much more ergonomic.

Reply:
actually, making from_dir positional is probably a better interface:

`garage resume data/foo/bar --from_epoch=last`

Reply:
I second on your last line. handy and pretty.